axum/CHANGELOG.md

```diff
@@ -11,9 +11,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   Use `axum::middleware::from_extractor` instead ([#1077])
 - **breaking:** Allow `Error: Into<Infallible>` for `Route::{layer, route_layer}` ([#924])
 - **breaking:** `MethodRouter` now panics on overlapping routes ([#1102])
+- **breaking:** Remove trailing slash redirects. Previously if you added a route
+  for `/foo` axum would redirect calls to `/foo/` to `/foo` (or vice versa for
+  `/foo/`). That is no longer supported and such requests will now be sent to
+  the fallback ([#1119])
```

Comment:
I think we should call out `route_with_tsr` here (not everybody is going to also check axum-extra's changelog).